docs/ipc.md

```diff
@@ -0,0 +1,55 @@
+# IPC
+
+This documentation will guide you through the different utils provided in Fendermint for the deployment of Fendermint-based IPC subnets. This docs are only focused on the infrastructure deployment, for an end-to-end walk through of spawning IPC subnets refer to the [IPC quickstart](https://github.com/consensus-shipyard/ipc/blob/main/docs/quickstart-calibration.md).
+
+## Deploy subnet bootstrap
+In order not to expose directly the network address information from validators, subnets leverage the use of bootstraps (or `seeds` in CometBFT parlance), for new nodes to discover peers in the network and connect to the subnet's validators. To run a bootstrap node you can run the following command from the root of the repo:
+```bash
+cargo make --makefile infra/Makefile.toml bootstrap
+```
+You'll see that by the end of the output, this command should output the network address of your bootstrap. You can use this endpoint to include this bootstrap node as a seed in the `seeds` configuration of CometBFT.
+```console
+[cargo-make] INFO - Running Task: cometbft-wait
+[cargo-make] INFO - Running Task: cometbft-node-id
+2b23b8298dff7711819172252f9df3c84531b1d9@172.26.0.2:26650
```

Comment:
This IP doesn't look like something the user can actually share with others on the network. By the looks of it it's the IP of the container inside the docker network, not the one visible externally over the internet.

Reply:
Of course the docs doesn't explicitly say sharing it with the world was the intention. It's not exactly clear whether these infra scripts are for local testing or to connect to a real network.

Reply:
> When I last asked @dnkolegov whether this IP was accessible at all even from the host, he said no, it's internal to Docker, so this sentence seems misleading:

hmm, I wasn't aware of this. How can we expose the bootstrap to the outside world? @dnkolegov, do you mind chiming in? 

> Why don't you print the host IP instead, e.g. the output of curl ifconfig.me ?

Maybe we should just output a placeholder and let the user choose the relevant IP? I though about `ifconfig.me` but what if this is being deployed over an AWS instance with all of their networking madness? But you are definitely right, and giving the internal IP for Docker is the most confusing option.

Reply:
I'd use `ifconfig.me` as a reflection of the host IP and mention it. I'm not sure what else it can be, but I'm fairly certain if someone deploys to AWS they will take more control over the process.

Reply:
Good point, will update accordingly, thanks!

Reply:
Cool, I see you changed the script. Maybe change the IP here as well to something operators with a keen eye would not recognise as a docker IP. They usually start with 172.x.x.x